_data/toc.yml

```diff
@@ -85,8 +85,8 @@
   - title: "Probabilistic PCA"
     url: "tutorials/11-probabilistic-pca"
 
-  - title: "Gaussian Processes"
-    url: "tutorials/12-gaussian-process"
+  - title: "GPLVM"
```

Comment:
I'd prefer the full name here, or at last "Gaussian Process LVM" or something.

Reply:
```suggestion
  - title: "Gaussian Process LVM"
```